new-docs/components/Header.vue

```diff
@@ -1,13 +1,8 @@
 <template>
-  <div class="w-full p-4 shadow-md">
+  <header class="w-full p-4 shadow-md">
     <div class="desktop-gutters flex items-center justify-center space-x-2">
-      <centrapay-logo class="w-12 h-12 fill-brand-accent" />
+      <lazy-icons-centrapay-logo class="w-12 h-12 fill-brand-accent" />
```

Comment:
https://v3.nuxtjs.org/guide/directory-structure/components#component-names and https://v3.nuxtjs.org/guide/directory-structure/components#dynamic-imports